contracts/FractalUsul.sol

```diff
@@ -1,86 +1,445 @@
-//SPDX-License-Identifier: MIT
```

Comment:
I think it might make sense to put this into the "usul" directory with the other Usul stuff (to be renamed whatever we go with)

also maybe Usul related interfaces within a usul/interfaces directory?  I'm not sure, would just like to have the "Usul" framework a self contained directory independent of any other contracts

Reply:
Yeah agreed, this makes sense too